python-client/hatchet_sdk/clients/dispatcher.py

```diff
@@ -166,15 +175,15 @@ def get_action_listener(self, req: GetActionListenerRequest) -> ActionListenerIm
             workerName=req.worker_name,
             actions=req.actions,
             services=req.services
-        ))
+        ), timeout=DEFAULT_REGISTER_TIMEOUT)
 
         # Subscribe to the worker
-        listener = self.client.Listen(WorkerListenRequest(
-            tenantId=self.tenant_id,
-            workerId=response.workerId,
-        ))
+        # listener = self.client.Listen(WorkerListenRequest(
+        #     tenantId=self.tenant_id,
```

Comment:
was this meant to be commented out?

Reply:
Nice catch - I actually meant to remove it, because the action listener handles its own `.Listen` invocation now.